mw-webapp/src/service/auth/constants.ts

```diff
@@ -0,0 +1,4 @@
+export const USER_IS_AUTH = "userIsAuth";
+export const PATH = "path";
+export const USER_REGISTERED = "true";
```

Comment:
1. For boolean values netter start name with Is
2. Do we register users, not just loginby Google?

Reply:
1. In this case, it indicates a flag as to whether the user is authorized or not. It will be strange to see the IS_USER_NOT_AUTH = "false"
2. ok, authorization